### ISSUE TITLE
Authorization code

### ISSUE THREAD

**Issue by NathanVss** [Wizcorp/phonegap-facebook-plugin#1249]
Hello everyone.

I read the doc and it is not mentioning how to get a authorization code from this plugin ? Is it possible ? 

Thanks.

**Comment by Dragsbaek** [comment]
I need this too

**Comment by NathanVss** [comment]
Actually I setup a implementation exchanging the access token returned by the plugin, it is not really safe until you use a HTTPS certificate ..

**Comment by Dragsbaek** [comment]
Yea, there should really be a way to just get the authorization code

**Comment by NathanVss** [comment]
I agree. Even the native Facebook SDK doesn't seem to provide this feature .. I don't know why.

**Comment by BHushanRathod** [comment]
is there any workaround to get Authorization Code?
